models/ProfileToFormRPartAInitialValues.ts

```diff
@@ -53,7 +53,7 @@ export function ProfileToFormRPartAInitialValues(
     completionDate: programme?.programmeCompletionDate ?? null,
     trainingGrade: "",
     startDate: programme?.startDate ?? null,
-    programmeMembershipType: programme?.programmeMembershipType ?? "",
+    programmeMembershipType: "",
```

Comment:
Perhaps this should really go into another PR - I noticed that the dropdown for new forms was being prepopulated with 'MILITARY'. On staging for some reason the dropdown appears blank which is why I missed it (though it does actually have the value if you proceed to review and submit the form!)